libfreerdp/core/update.c

```diff
@@ -2978,12 +2983,15 @@ void update_free(rdpUpdate* update)
 		}
 
 		free(update->secondary);
+		update->secondary = NULL;
```

Comment:
What should this be good for?

Reply:
prevent possibility for use after free and double free corruption

Reply:
Remove, unnecessary